src/igvf_and_terra_api_tools.py

```diff
@@ -0,0 +1,88 @@
+import firecloud.api as fapi
+import subprocess
+import pandas as pd
+import io
+import dumper
+from igvf_client import Configuration
+from igvf_client import ApiClient
+from igvf_client import IgvfApi
+from igvf_utils.connection import Connection
+import os
+
+
+def get_igvf_auth_and_api(igvf_site: str = 'sandbox'):
+    """Set up IGVF data portal access and set up IGVF python client api.
+    """
+    if igvf_site == 'production':
+        host = 'https://api.data.igvf.org'
+    elif igvf_site == 'sandbox':
+        host = 'https://api.sandbox.igvf.org'
+    config = Configuration(
+        access_key=os.environ["IGVF_API_KEY"],
+        secret_access_key=os.environ["IGVF_SECRET_KEY"],
+        host=host
+    )
+    client = ApiClient(config)
+    return IgvfApi(client)
+
+
+def get_igvf_utils_connection(igvf_utils_mode: str = 'sandbox'):
+    return Connection(igvf_mode=igvf_utils_mode, submission=True)
+
+
+fapi._set_session()
```

Comment:
Don't know much about firecloud api. Is _set_session() an internal function? If you really need to use it here Can this move to the top of the file?